app/src/main/res/layout/activity_login.xml

```diff
@@ -1,79 +1,150 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:paddingLeft="@dimen/activity_horizontal_margin"
-    android:paddingRight="@dimen/activity_horizontal_margin"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:paddingTop="@dimen/activity_vertical_margin"
     android:paddingBottom="@dimen/activity_vertical_margin"
-    android:background="#F8F7F3">
+    android:background="@color/offWhite">
 
     <ImageView
         android:id="@+id/logo"
         android:contentDescription="@string/logo_description"
         android:layout_centerHorizontal="true"
+        android:adjustViewBounds="true"
         android:layout_height="wrap_content"
+        android:layout_width="match_parent"
+        android:src="@drawable/home_logo" />
+
+    <xyz.whereuat.whereuat.ui.views.LatoTextView
+        android:id="@+id/logo_text"
         android:layout_width="wrap_content"
-        android:src="@drawable/logo" />
+        android:layout_height="wrap_content"
+        app:lato_font="light"
+        android:textSize="@dimen/logo_font_size"
+        android:text="@string/app_name"
+        android:textColor="@color/coolRed"
+        android:layout_below="@+id/logo"
+        android:layout_centerInParent="true"
+        android:layout_marginBottom="@dimen/section_margin"/>
 
     <RelativeLayout
         android:id="@+id/account_request_section"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
-        android:layout_below="@+id/logo">
-        <EditText
-            android:id="@+id/phone_number_input"
-            android:background="@drawable/textinputborder"
-            android:focusable="true"
-            android:gravity="center"
-            android:hint="@string/phone_number_input_text"
-            android:inputType="phone"
-            android:layout_alignParentLeft="true"
-            android:layout_alignParentRight="true"
-            android:layout_centerHorizontal="true"
-            android:layout_height="@dimen/text_input_height"
-            android:layout_width="wrap_content"
-            android:textColor="@color/offBlack" />
-        <Button
-            android:id="@+id/account_request_btn"
-            android:background="@color/coolRed"
-            android:layout_alignParentBottom="true"
-            android:layout_centerHorizontal="true"
+        android:layout_below="@+id/logo_text"
+        android:layout_marginLeft="@dimen/activity_horizontal_margin"
+        android:layout_marginRight="@dimen/activity_horizontal_margin">
+        <RelativeLayout
```

Comment:
Does this need to be wrapped in a RelativeLayout too? Why not just make one RelativeLayout with a TextView, EditText, and Button inside?